splunk_add_on_ucc_framework/generators/globalConfig_generator/globalConfig_generator.py

```diff
@@ -0,0 +1,35 @@
+#
+# Copyright 2025 Splunk Inc.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+from typing import Any, Dict, Union, NoReturn
+from ..file_generator import FileGenerator
+
+
+class GlobalConfigGenerator(FileGenerator):
```

Comment:
Can `MinimalGlobalConfig` inherit `FileGenerator` directly without this middleware?

Reply:
I did think about it, but didn't proceed with it as I thought we might move the `ucc-gen init` command to generate globalConfig with this hierarchy, just to keep everything in same umbrella.

Reply:
I think we can make it much simpler but inheriting directly, this way we can avoid having `tests/unit/generators/globalConfig_generator/test_globalConfig_generator.py` at all.

We can always refactor it later if we need to bring in another feature.